go/vt/sqlparser/sql.y

```diff
@@ -5646,6 +5646,7 @@ window_partition_clause_opt:
 
 sql_id_opt:
   {
+    $$ = IdentifierCI{}
```

Comment:
Is an empty block in the parser in general a signal for a bug? And if so, do we have more cases like this?

Reply:
@systay tried to simplify the query to a minimum but that was the only way to trigger the bug AFAIK